tests/scripts/test_config.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+. "$(dirname "$(realpath $0)")/config.sh"
+
+good_settings="good_settings.toml"
+bad_settings="bad_settings.toml"
+good_cli="$PULP_CLI --config $good_settings"
```

Comment:
We can now provide the settings via giving a `XDG_CONFIG_HOME` pointing in the right place.
So we can get rid of this `$PULP_CLI` variable. WDYT?

Reply:
I don't know. Right now my pain point is that I have to supply `--base_url` every time I use the CLI since this PR isn't merged.

Reply:
I created a task for your suggestion: https://pulp.plan.io/issues/7727